### PR TITLE
chore: update metadata built-in list

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/control-plane/backend/config/dictionary.json [hunk 1]
```diff
@@ -86,78 +86,120 @@
...
-            "en-US": "The app is launched",
-            "zh-CN": "应用启动"
+            "en-US": "Every time the app goes to visible",
+            "zh-CN": "每次App变为可见时"
           }
         },
         {
           "name": "_app_end",
           "description": {
             "en-US": "The app is closed",
```
Comment (zxkane) [inline src/control-plane/backend/config/dictionary.json:124]: It looks inconsistent with Chinese. Should be `the app is invisiable` or hidden? @zhu-xiaowei

File: src/control-plane/backend/config/dictionary.json [hunk 2]
```diff
@@ -194,6 +236,38 @@
...
+        {
+          "name": "_page_title",
+          "dataType": "String",
+          "description": {
+            "en-US": "The page title",
+            "zh-CN": "页面标题"
+          }
+        },
+        {
+          "name": "",
```
Comment (zxkane) [inline src/control-plane/backend/config/dictionary.json:248]: why is it empty?

File: src/control-plane/backend/config/dictionary.json [hunk 3]
```diff
@@ -202,13 +276,197 @@
...
             "en-US": "The screen unique ID",
             "zh-CN": "屏幕唯一ID"
           }
+        },
+        {
+          "name": "_previous_screen_name",
+          "dataType": "String",
+          "description": {
+            "en-US": "The previous screen name",
+            "zh-CN": ""
```
Comment (zxkane) [inline src/control-plane/backend/config/dictionary.json:300]: zh-CN description is missing

File: src/control-plane/backend/config/dictionary.json [hunk 4]
```diff
@@ -202,13 +276,197 @@
...
+          }
+        },
+        {
+          "name": "_previous_screen_id",
+          "dataType": "String",
+          "description": {
+            "en-US": "The previous screen ID",
+            "zh-CN": ""
```
Comment (zxkane) [inline src/control-plane/backend/config/dictionary.json:308]: zh-CN description is missing

File: src/control-plane/backend/config/dictionary.json [hunk 5]
```diff
@@ -202,13 +276,197 @@
...
+          }
+        },
+        {
+          "name": "_previous_screen_unique_id",
+          "dataType": "String",
+          "description": {
+            "en-US": "The previous screen unique ID",
+            "zh-CN": ""
```
Comment (zxkane) [inline src/control-plane/backend/config/dictionary.json:316]: zh-CN description is missing

**Review by zxkane — CHANGES_REQUESTED** [review]

**Review by zxkane — APPROVED** [review]

File: src/control-plane/backend/config/dictionary.json [hunk 6]
```diff
@@ -86,361 +86,643 @@
...
             "zh-CN": "当前用户生命周期价值"
           }
         },
         {
           "name": "_user_first_touch_timestamp",
-          "dataType": "String",
+          "dataType": "string",
           "description": {
             "en-US": "The timestamp of the first touch",
             "zh-CN": "首次触摸时间戳"
```
Comment (zhu-xiaowei) [inline src/control-plane/backend/config/dictionary.json:728]: 用户当前设备首次访问的时间戳

**Pull request merged**